### PR TITLE
Create code.org/administrators page

### PR DESCRIPTION
**Pull request by kelbyhawn**
Creates a new https://code.org/administrators page.
- Page is fully responsive
- Includes the new `curriculum_pathway.haml` view
- There are a few `TODO` comments I'll follow up with in the near future

**Jira ticket:** [ACQ-1323](https://codedotorg.atlassian.net/browse/ACQ-1323)

----

localhost code org_3000_administrators

### CODE REVIEW

File: pegasus/sites.v3/code.org/styles_min/040-page.css [hunk 1]
```diff
@@ -263,6 +263,16 @@ input[type=submit] {
     &:hover {
       background: var(--neutral_dark10);
     }
+
+    &.white {
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/styles_min/040-page.css:267]: Adds white styles for the `.link-button.secondary` class.

File: pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss [hunk 2]
```diff
@@ -1275,6 +1306,10 @@ section.quote {
   border-radius: $border-radius;
   padding: 2rem;
 
+  &.no-bg {
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss:1309]: Adds a class to remove the background on a Subscribe form section.

**Comment by dmcavoy** [comment]
Screenshot 2024-01-26 at 2 17 10 PM

Looking at some of the existing links since code.org/administrators currently redirects to code.org/districts. Here are a couple clean ups I think would be good

code.org/district should go to code.org/districts
code.org/interest-form should go to code.org/districts
Delete the redirect for /K5-partnerships
code.org/district/partners should probably go to the new district recognition page
code.org/qualify - lets delete it
code.org/term - lets delete it

Thoughts?

**Review by TurnerRiley — APPROVED** [review]
LGTM!

**Pull request merged**